website/site/content/docs/custom-widgets.md

```diff
@@ -19,16 +19,21 @@ However, although possible, it may be cumbersome or even impractical to add a Re
 
 Register a custom widget.
 
-```js
-// Using global window objectCMS.registerWidget(name, control, [preview])// Using npm module importimport CMS from 'netlify-cms'CMS.registerWidget(name, control, [preview])
+```js
+// Using global window object
+CMS.registerWidget(name, control, [preview]);
+
+// Using npm module import
+import CMS from 'netlify-cms';
+CMS.registerWidget(name, control, [preview]);
 ```
 
 **Params:**
 
 | Param       | Type                      | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                 |
 | ----------- | ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `name`      | string                    | Widget name, allows this widget to be used via the field `widget` property in config                                                                                                                                                                                                                                                                                                                                                                                        |
-| `control`   | React.Component | string  | <ul><li>React component that renders the control, receives the following props: <ul><li>**value:** Current field value</li><li>**onChange:** Callback function to update the field value</li></ul></li><li>Name of a registered widget whose control should be used (includes built in widgets).</li></ul>                                                                                                                                                                  |
+| `control`   | React.Component \| string  | <ul><li>React component that renders the control, receives the following props: <ul><li>**value:** Current field value</li><li>**onChange:** Callback function to update the field value</li></ul></li><li>Name of a registered widget whose control should be used (includes built in widgets).</li></ul>                                                                                                                                                                  |
```

Comment:
What does `React.Component | string` mean?

Reply:
@erquhart It looks like you were the person who first added this line.
Is it possible the ` | string` part is a typo carried over by copy-pasting the line before?

Reply:
`React.Component | string` means it can be a React component or a string.